app/assets/javascripts/upload_setup.js

```diff
@@ -11,6 +11,12 @@ var uploadSetup = function (previewTemplate) {
       window.alert(msg)
     },
     success: function (f) {
+      $('.button--next').removeAttr('disabled')
+
+      var button = $('#click-to-upload')
+      var buttonHtml = button.html()
+      button.html(buttonHtml.replace('Upload documents', 'Upload more documents'))
```

Comment:
this js isn't terribly complex, but it would be nice if we could test this behavior somehow so specs would fail it we mess up the js in the future. thoughts?